core-build/gulp-core-build-sass/tsconfig.json

```diff
@@ -1,3 +1,8 @@
 {
-  "extends": "./node_modules/@microsoft/rush-stack-compiler-3.2/includes/tsconfig-node.json"
+  "extends": "./node_modules/@microsoft/rush-stack-compiler-3.2/includes/tsconfig-node.json",
+  "compilerOptions": {
+    "types": [
+      "mocha"
```

Comment:
@iclanton Did we not convert **gcb-sass** to use Jest yet?

Reply:
I don't think any of the GCB projects are on Jest yet. It'd be great to use Jest in this PR, though. I think this is actually the first test being added for GCB-sass, so that should be an easy change.

Reply:
Now using Jest.